test/controllers/api/base_controller_test.rb

```diff
@@ -0,0 +1,12 @@
+require_relative '../../test_helper'
+
+SingleCov.covered!
+
+describe Api::BaseController do
+  describe 'Doorkeeper Auth Status' do
+    subject { Api::BaseController }
+    it 'is allowed' do
```

Comment:
maybe also do `is allowed for subclass` to make sure it's inherited properly and `is not allowed for superclass`